app/konnect/org-management/audit-logging/verify-signatures.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Verify audit log signatures
+content_type: how-to
+badge: enterprise
+beta: true
+---
+
+{{site.konnect_short_name}} uses an [ED25519 signature](https://ed25519.cr.yp.to/) on the audit logs it produces.
+
+Audit logs can be exported in two different formats, CEF and JSON. 
+Calculating the signature is slightly different for these formats.
+
+## Verify a signature
+
+1. Retrieve the public key from the audit log JWKS endpoint:
+
+    ```sh
+    curl -i -X GET https://us.api.konghq.com/v2/audit-log-webhook/jwks.json \
+    --header "Authorization: Bearer TOKEN"
+    ```
+
+    The response should look something like this:
+
+    ```json
+    {
+        "keys": [
+            {
+                "alg": "EdDSA",
+                "crv": "Ed25519",
+                "kid": "1d4608c22e448672d5386b4071b70442as45c58265",
+                "kty": "OKP",
+                "x": "aFNAu9QEQhiunrGuyS14ePHzoOb2vash783p1-_Nrc3M"
+            }
+        ]
+    }
+    ```
+1. Extract the signature of the entry to calculate the signature to match. 
+
+1. Decode the Base64 URL-encoded signature into bytes.
```

Comment:
It looks like both the signature and the public key are Base64 URL-encoded. So both will need to be decoded into bytes.